### PR TITLE
NFDiv-2659 and NFDiv-2751 Hub Page After Switching To Sole Intended FO

### PR DESCRIPTION
**Pull request by BWeeks-JusticeGovUK**
https://tools.hmcts.net/jira/browse/NFDIV-2659
https://tools.hmcts.net/jira/browse/NFDIV-2751

### CODE REVIEW

File: src/main/steps/applicant1/hub-page/joint/content.ts [hunk 1]
```diff
@@ -137,6 +138,13 @@ const en = ({ isDivorce, userCase, partner, isApplicant2 }: CommonContent) => ({
     }.`,
     line2: 'You should receive an email within 2 working days, confirming whether the final order has been granted.',
   },
+  intendToSwitchToSoleFinalOrder: {
+    line1: `The court has notified your ${partner} by email that you are intending to apply for a final order as a sole applicant.`,
+    // Correct this offset.  Base on now()?  Probably also need new config.date param for switch to sole FO offset.
```
Comment (benouaer) [inline src/main/steps/applicant1/hub-page/joint/content.ts:143]: is the comment still needed?

File: src/main/steps/applicant1/hub-page/joint/content.ts [hunk 2]
```diff
@@ -137,6 +138,13 @@ const en = ({ isDivorce, userCase, partner, isApplicant2 }: CommonContent) => ({
...
+    line2: `You will be able to apply for a final order from ${getFormattedDate(
+      dayjs(now()).add(config.get('dates.finalOrderSubmittedOffsetDays'), 'day')
+    )}. You will receive an email to remind you.`, // Is this always an email or does this need clarification?
```
Comment (benouaer) [inline src/main/steps/applicant1/hub-page/joint/content.ts:146]: is the comment still needed?

File: src/main/steps/applicant1/hub-page/joint/content.ts [hunk 3]
```diff
@@ -265,6 +273,13 @@ const cy: typeof en = ({ isDivorce, userCase, partner, isApplicant2 }: CommonCon
       reference: 'Final-Order-Granted',
     },
   },
+  intendToSwitchToSoleFinalOrder: {
+    line1: `Mae'r llys wedi hysbysu eich ${partner} drwy e-bost eich bod yn bwriadu gwneud cais am orchymyn terfynol fel unig geisydd.`,
+    // Correct this offset.  Base on now()?  Probably also need new config.date param for switch to sole FO offset.
```
Comment (benouaer) [inline src/main/steps/applicant1/hub-page/joint/content.ts:278]: same as above

File: src/main/steps/applicant1/hub-page/joint/content.ts [hunk 4]
```diff
@@ -265,6 +273,13 @@ const cy: typeof en = ({ isDivorce, userCase, partner, isApplicant2 }: CommonCon
...
+    line2: `Byddwch yn gallu gwneud cais am orchymyn terfynol o ${getFormattedDate(
+      dayjs(now()).add(config.get('dates.finalOrderSubmittedOffsetDays'), 'day')
+    )}. Byddwch yn cael e-bost i'ch atgoffa.`, // Is this always an email or does this need clarification?
```
Comment (benouaer) [inline src/main/steps/applicant1/hub-page/joint/content.ts:281]: same as above

**Review by benouaer — APPROVED** [review]

File: src/main/steps/applicant1/hub-page/joint/content.ts [hunk 5]
```diff
@@ -323,5 +369,6 @@ export const generateContent: TranslationFn = content => {
     hasApplicantAppliedForFinalOrderFirst,
     isFinalOrderCompleteState,
     finalOrderEligibleAndSecondInTimeFinalOrderNotSubmittedWithin14Days,
+    withinSwitchToSoleFinalOrderIntentionNotificationWindow,
```
Comment (danielbe11) [inline src/main/steps/applicant1/hub-page/joint/content.ts:372]: We don't need this here because it's not being used within the njk file

**Pull request merged**